src/bundle/Resources/public/scss/mixins/_drag-and-drop.scss

```diff
@@ -307,7 +307,28 @@
     }
 
     &--invalid {
-        border-color: $ibexa-color-danger;
+        #{$self}__content {
+            border-color: $ibexa-color-danger;
+
+            * {
+                color: $ibexa-color-danger;
+            }
```

Comment:
Why this way?
Can you change color only for classes thats need it?

Reply:
I copied it from &--highlighted, but all I need to do is set the color to #{$self}__content. This has already been changed